<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background: rgba(50, 50, 50, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>811</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>340</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_lights">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>370</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Освещение</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDetail">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Деталь</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="typeDetail">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addDetail">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeDetail">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>300</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeDetail">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addLight">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>400</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeLight">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>610</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Перемещение</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>96</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="dMoveY">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="dMoveX">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelDMoveX">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDMoveY">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelDMoveZ">
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="dMoveZ">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Масштабирование</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>96</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QLineEdit" name="dScaleY">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="dScaleX">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelDScaleX">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDScaleY">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelDScaleZ">
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="dScaleZ">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Поворот</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>96</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="1">
          <widget class="QLineEdit" name="dRotateY">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="dRotateX">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelDRotateX">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDRotateY">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelDRotateZ">
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="dRotateZ">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>440</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Перемещение</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="1">
          <widget class="QLineEdit" name="lMoveY">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lMoveX">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelLMoveX">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelLMoveY">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelLMoveZ">
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lMoveZ">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Мощность</string>
     </attribute>
     <widget class="QLineEdit" name="lPower">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
